<launch>
  <node name="takeoff_landing_planner" pkg="core_takeoff_landing_planner" type="takeoff_landing_planner" output="screen">
    <param name="execute_target" type="double" value="20." />

    <param name="takeoff_height" type="double" value="50.0" />
    <param name="takeoff_landing_velocity" type="double" value="1" />
    <param name="takeoff_acceptance_distance" type="double" value="0.3" />
    <param name="takeoff_acceptance_time" type="double" value="1.0" />
    <param name="landing_stationary_distance" type="double" value="0.02" />
    <param name="landing_acceptance_time" type="double" value="5.0" />

    <param name="takeoff_path_roll" type="double" value="0" /> <!-- degrees -->
    <param name="takeoff_path_pitch" type="double" value="0" /> <!-- degrees -->
    <param name="takeoff_path_relative_to_orientation" type="bool" value="false" />

  </node>
</launch>